packages/apm/src/integrations/tracing.ts

```diff
@@ -283,6 +283,52 @@ export class Tracing implements Integration {
     });
   }
 
+  /**
+   * Returns a new Transaction either continued from sentry-trace meta or a new one
+   */
+  private static _getNewTransaction(hub: Hub, transactionContext: TransactionContext): Transaction {
+    let traceId;
+    let parentSpanId;
+    let sampled;
+
+    const header = Tracing._getMeta('sentry-trace');
+    if (header) {
+      const span = SpanClass.fromTraceparent(header);
+      if (span) {
+        traceId = span.traceId;
+        parentSpanId = span.parentSpanId;
+        sampled = span.sampled;
+      }
+      Tracing._log(
```

Comment:
This log will be incorrect if `span` is falsy. Should be moved inside `span` or corrected logically.